### PR TITLE
Add Semaphore Basics section

### PR DESCRIPTION
**Pull request by ivanaur**
Based on:
- https://3.basecamp.com/3093216/buckets/2361210/todos/2460848019
and
- https://3.basecamp.com/3093216/buckets/260166/todos/2464456591

### CODE REVIEW

File: docs/guided-tour/getting-started.md [hunk 1]
```diff
@@ -62,4 +221,4 @@ customizing it for your use case.
 
 If you want to find out more about the product and participate in a live Q&A
```
Comment (Ivana-Markovic) [inline docs/guided-tour/getting-started.md]: I am not sure what will be done re. Semaphore Class but I'll check with Darko. It is possible that we will need to reword this part. Everything else is 👌

Comment (markoa) [reply docs/guided-tour/getting-started.md]: Remove this paragraph. The Class is on hold for a while now.

**Comment by shiroyasha** [comment]
@Ivana-Markovic @markoa @ivanaur This has been open for 1.5 months. What is the state on this PR?

**Comment by Ivana-Markovic** [comment]
Darko changed his mind at the end. I do not think we will add this to our docs in this form. I am closing this.

**Pull request closed without merging**